translations/harbour-fernweh_it.ts

```diff
@@ -0,0 +1,435 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1">
+<context>
+    <name>AboutPage</name>
+    <message>
+        <source>About Fernweh</source>
+        <translation type=A proposito di Fernweh></translation>
```

Comment:
```suggestion
        <translation>A proposito di Fernweh</translation>
```

Please change this for all translations as this is invalid XML and won't work.